Sources/MongoSwift/ChangeStream.swift

```diff
@@ -137,11 +142,18 @@ public class ChangeStream<T: Codable>: Sequence, IteratorProtocol {
      *   - `UserError.invalidArgumentError` if the `mongoc_change_stream_t` was created with invalid options.
      */
     internal init(stealing changeStream: OpaquePointer,
+                  options: ChangeStreamOptions? = nil,
                   client: MongoClient,
                   connection: Connection,
                   session: ClientSession? = nil,
                   decoder: BSONDecoder) throws {
-        self.resumeToken = ResumeToken(resumeToken: [])
+        if let options = options {
```

Comment:
nit: these `if` statements can be combined using optional chaining: `if let resumeAfter = options?.resumeAfter`